HW1/P6/P6.txt

```diff
@@ -1 +1,12 @@
-# Your discussion here
+Ten randomly generated phrases:
+
+1.) indeed, that baes like a man of his own hand did make, Breathed forth the stocks! You stubborn ancient knave,
```

Comment:
Yes, I am always taking the most likely next word. That is how I initially understood assignment. After rereading it now it seems that you are right, I probably should have done a weighted sampling instead.